lib/utils/progress.go

```diff
@@ -144,6 +146,17 @@ func (p *ConsoleProgress) PrintCurrentStep(message string, args ...interface{})
 	PrintStep(entry.current, p.steps, entry.message)
 }
 
+// PrintSubStep outputs the message as a sub-step of the current step
+func (p *ConsoleProgress) PrintSubStep(message string, args ...interface{}) {
+	message = fmt.Sprintf(message, args...)
```

Comment:
Looks like this could share some code with PringCurrentStep:
```go
func (p *ConsoleProgress) PrintSubStep(message string, args ...interface{}) 
	message = p.updateEntry(message, args...)
	fmt.Fprintf(os.Stdout, "\t%v\n", message)
```